src/model.ts

```diff
@@ -65,6 +65,25 @@ export interface IColumnDesc {
    * css class to append to elements of this column
    */
   cssClass?: string;
+  /**
```

Comment:
why are you manipulting the generic column description? Those additional attributes are just valid for your cases not in the generic case?

None of the other columns are doing that